pkg/setting/setting.go

```diff
@@ -1650,6 +1654,11 @@ func readAuthSettings(iniFile *ini.File, cfg *Cfg) (err error) {
 	cfg.BasicAuthEnabled = authBasic.Key("enabled").MustBool(true)
 	cfg.BasicAuthStrongPasswordPolicy = authBasic.Key("password_policy").MustBool(false)
 
+	// passwordless auth
+	passwordlessSection := iniFile.Section("auth.passwordless")
+	cfg.PasswordlessEnabled = passwordlessSection.Key("enabled").MustBool(false)
+	cfg.PasswordlessCodeExpiration = passwordlessSection.Key("code_expiration").MustDuration(1 * time.Minute)
```

Comment:
Nit: prefer to create a structure for all passwordless settings and use that instead. see https://github.com/grafana/grafana/blob/df4a5aeaf1d5fea40b5688416f7a4260ecdf6cf8/pkg/setting/setting_jwt.go as an example